spring-pulsar/src/main/java/org/springframework/pulsar/core/DefaultPulsarConsumerFactory.java

```diff
@@ -77,10 +79,19 @@ public Consumer<T> createConsumer(Schema<T> schema, BatchReceivePolicy batchRece
 		final Map<String, Object> properties = new HashMap<>(this.consumerConfig);
 		properties.putAll(propertiesToOverride);
 
+		DeadLetterPolicy deadLetterPolicy = null;
```

Comment:
Thanks for adding link to underlying cause of this workaround. I think adding a one-liner (similar to your [review comment](https://github.com/spring-projects-experimental/spring-pulsar/pull/88#discussion_r963599760) may be helpful as it may not be clear (in say, 6mos. from now) that the code on 91 (after loadConf) is also part of the workaround.